index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta name="keywords" content="css, html5, css framework"/>
     <meta name="description" content="CSS Framework for Dummies"/>
-    <link rel="stylesheet" href="tacit.min.css"/>
+    <link rel="stylesheet" href="dist/tacit-css-0.0.0.min.css"/>
```

Comment:
@akshayvinchurkar1 this isn't right, we shouldn't point to `0.0.0` version here.